src/rebuild.ts

```diff
@@ -220,6 +220,29 @@ export function buildNodeWithSN(
   return node as INode;
 }
 
+function sideEffects(idNodeMap: idNodeMap) {
```

Comment:
we need these side effects because apply this scroll dimension to the element during rebuild does not work?

Reply:
you are right